lib/linter.js

```diff
@@ -908,6 +908,7 @@ module.exports = class Linter {
 
         this.rules = new Rules();
         this.environments = new Environments();
+        this._logger = debug;
```

Comment:
Just for my own understanding - is the reasoning for creating this property to make it testable?